internal/daemon/cluster/handlers/upstream_message_service_controller.go

```diff
@@ -19,7 +19,11 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-var upstreamMessageHandler sync.Map
+var upstreamMessageHandler *sync.Map
+
+func init() {
+	upstreamMessageHandler = new(sync.Map)
```

Comment:
Can't we do this in the declaration instead of during init?

Reply:
Ah yeah fair, fixed